std/format.d

```diff
@@ -515,7 +515,7 @@ uint formattedWrite(Writer, Char, A...)(Writer w, in Char[] fmt, A args)
             // means: get width as a positional parameter
             auto index = cast(uint) -spec.width;
             assert(index > 0);
-            auto width = to!(typeof(spec.width))(getNthInt(index - 1, args));
+            auto width = getNthInt!"integer width"(index - 1, args);
```

Comment:
Yes, `getNthInt` already converts the argument to `int`, and `spec.width` is `int`. Same for the other spec fields used with `getNthInt`.